sources/packages/api/src/route-controllers/institution/institution.institutions.controller.ts

```diff
@@ -84,17 +84,17 @@ export class InstitutionInstitutionsController extends BaseController {
   }
 
   /**
-   * Create institution on Institution setup process.
-   * @param payload
-   * @param userToken
-   * @returns Primary identifier of the created resource.
+   * Creates an institution during institution setup process when the
+   * institution profile and the user are create and associated altogether.
```

Comment:
Small typo here. Please change "create" to "created".

Reply:
Change in 2 points with the same mistake. I will update the PR soon.

Reply:
Changed.